src/__tests__/actionTests/profileActionTests.js

```diff
@@ -0,0 +1,184 @@
+
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import getProfile from '../../redux/actions/profileActions';
+import updateProfile, { uploadImage, updateSpinnerStatus } from '../../redux/actions/updateProfileActions';
+import {
+  FETCH_PROFILE_SUCCESS, SPINNER_STATUS, FETCH_PROFILE_ERROR, UPDATE_PROFILE_SUCCESS, UPDATE_PROFILE_ERROR, UPLOAD_IMAGE_SUCCESS, UPLOAD_IMAGE_ERROR,
+} from '../../redux/actionTypes/profileActionTypes';
+
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+let store;
+jest.fn();
+describe('Profile Actions Test Suite', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+
+  afterEach(() => {
+    moxios.uninstall();
+  });
+
+  it('Should dispatch FETCH_PROFILE_SUCCESS', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          message: 'Success',
+        },
+      });
+    });
+    const expectedActions = [
+      {
+        spinner: false,
+        type: SPINNER_STATUS,
+      },
+      {
+        payload: {
+          message: 'Success',
+        },
+        type: FETCH_PROFILE_SUCCESS,
+      }];
```

Comment:
this one too, and any other data that can be placed in the mock data without complicating your work, But also there are others you can just leave it in the tests for some reasons.